From ceb17889bb64b964be7a5439344cc243f0498512 Mon Sep 17 00:00:00 2001
From: Peter Tribble <peter.tribble@gmail.com>
Date: Sun, 22 May 2022 15:37:28 +0100
Subject: [PATCH] 14702 Some cases of he in the manual should be the Reviewed
 by: Robert Mustacchi <rm@fingolfin.org> Reviewed by: Andrew Stormont
 <andyjstormont@gmail.com> Approved by: Dan McDonald <danmcd@mnx.io>

---
 usr/src/man/man3head/math.h.3head       | 11 ++---------
 usr/src/man/man3sip/sip_stack_init.3sip | 13 ++-----------
 usr/src/man/man3xnet/freeaddrinfo.3xnet | 11 ++---------
 3 files changed, 6 insertions(+), 29 deletions(-)

diff --git a/usr/src/man/man3head/math.h.3head b/usr/src/man/man3head/math.h.3head
index 78255659c5..f5894218cb 100644
--- a/usr/src/man/man3head/math.h.3head
+++ b/usr/src/man/man3head/math.h.3head
@@ -43,18 +43,15 @@
 .\" Copyright 1989 AT&T.
 .\" Copyright (c) 2004, Sun Microsystems, Inc.  All Rights Reserved.
 .\"
-.TH MATH.H 3HEAD "Aug 11, 2004"
+.TH MATH.H 3HEAD "May 19, 2022"
 .SH NAME
 math.h, math \- mathematical declarations
 .SH SYNOPSIS
-.LP
 .nf
 \fB#include <math.h>\fR
 .fi
 
 .SH DESCRIPTION
-.sp
-.LP
 The <\fBmath.h\fR> header includes definitions for the following types:
 .sp
 .ne 2
@@ -373,7 +370,7 @@ the program. If a macro definition is suppressed or a program defines an
 identifier with the name \fBmath_errhandling\fR, the behavior is undefined.
 .sp
 .LP
-The <\fBmath.h\fR> header defines he following external variable:
+The <\fBmath.h\fR> header defines the following external variable:
 .sp
 .in +2
 .nf
@@ -386,8 +383,6 @@ extern int signgam;
 The <\fBmath.h\fR> header defines the structure and constants used by the
 \fBmatherr\fR(3M) error-handling mechanisms.
 .SH ATTRIBUTES
-.sp
-.LP
 See \fBattributes\fR(7) for descriptions of the following attributes:
 .sp
 
@@ -402,8 +397,6 @@ Interface Stability	Standard
 .TE
 
 .SH SEE ALSO
-.sp
-.LP
 .BR Intro (3),
 .BR fenv.h (3HEAD),
 .BR limits.h (3HEAD),
diff --git a/usr/src/man/man3sip/sip_stack_init.3sip b/usr/src/man/man3sip/sip_stack_init.3sip
index f363accd02..0610743135 100644
--- a/usr/src/man/man3sip/sip_stack_init.3sip
+++ b/usr/src/man/man3sip/sip_stack_init.3sip
@@ -3,11 +3,10 @@
 .\" The contents of this file are subject to the terms of the Common Development and Distribution License (the "License").  You may not use this file except in compliance with the License.
 .\" You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE or http://www.opensolaris.org/os/licensing.  See the License for the specific language governing permissions and limitations under the License.
 .\" When distributing Covered Code, include this CDDL HEADER in each file and include the License file at usr/src/OPENSOLARIS.LICENSE.  If applicable, add the following below this CDDL HEADER, with the fields enclosed by brackets "[]" replaced with your own identifying information: Portions Copyright [yyyy] [name of copyright owner]
-.TH SIP_STACK_INIT 3SIP "Jan 23, 2007"
+.TH SIP_STACK_INIT 3SIP "May 19, 2022"
 .SH NAME
 sip_stack_init \- initializes SIP stack
 .SH SYNOPSIS
-.LP
 .nf
 \fBcc\fR [ \fIflag\fR ... ] \fIfile\fR ... -lsip [ \fIlibrary\fR ... ]
 #include <sip.h>
@@ -16,7 +15,6 @@ sip_stack_init \- initializes SIP stack
 .fi
 
 .SH DESCRIPTION
-.LP
 The \fBsip_stack_init()\fR function is used to initialize the \fBSIP\fR stack.
 The stack can be initialized by a process only once. Any shared library that is
 linked with a main program or another library that has already initialized the
@@ -59,7 +57,6 @@ all.
 .RE
 
 .SS "Upper Layer Registrations"
-.LP
 These include callbacks that are invoked to deliver incoming messages or error
 notification.
 .sp
@@ -175,7 +172,6 @@ If these callback routines are registered, the stack invokes
 .RE
 
 .SS "Connection Manager Interface"
-.LP
 The connection manager interfaces must be registered by the application to
 provide I/O related functionality to the stack. These interfaces act on a
 connection object that is defined by the application. The application registers
@@ -321,7 +317,6 @@ retransmit interval - default \fB32\fR \fBsecs\fR).
 .RE
 
 .SS "Custom \fBSIP\fR headers"
-.LP
 In addition to the \fBSIP\fR headers supported by the stack, an application can
 optionally provide a table of custom headers and associated parsing functions.
 The table is an array with an entry for each header. If the table includes
@@ -354,7 +349,7 @@ typedef struct header_function_table {
 \fBheader_name\fR
 .ad
 .RS 27n
-The full name of the header. The application must ensure that he name does not
+The full name of the header. The application must ensure that the name does not
 conflict with existing headers. If it does, the one registered by the
 application takes precedence.
 .RE
@@ -410,7 +405,6 @@ application.
 .RE
 
 .SH RETURN VALUES
-.LP
 On success \fBsip_stack_init()\fR returns \fB0\fR. Otherwise, the function
 returns the error value.
 .sp
@@ -418,7 +412,6 @@ returns the error value.
 The value of \fBerrno\fR is not changed by these calls in the event of an
 error.
 .SH ERRORS
-.LP
 On failure, the \fBsip_stack_init()\fR function returns the following error
 value:
 .sp
@@ -432,7 +425,6 @@ missing.
 .RE
 
 .SH ATTRIBUTES
-.LP
 See \fBattributes\fR(7) for descriptions of the following attributes:
 .sp
 
@@ -449,6 +441,5 @@ MT-Level	MT-Safe
 .TE
 
 .SH SEE ALSO
-.LP
 .BR libsip (3LIB),
 .BR sockaddr (3SOCKET)
diff --git a/usr/src/man/man3xnet/freeaddrinfo.3xnet b/usr/src/man/man3xnet/freeaddrinfo.3xnet
index 168d285faa..3fddfd5c6f 100644
--- a/usr/src/man/man3xnet/freeaddrinfo.3xnet
+++ b/usr/src/man/man3xnet/freeaddrinfo.3xnet
@@ -43,11 +43,10 @@
 .\" Copyright (c) 2001, The IEEE and The Open Group.  All Rights Reserved.
 .\" Portions Copyright (c) 2003, Sun Microsystems, Inc. All Rights Reserved.
 .\"
-.TH FREEADDRINFO 3XNET "Feb 25, 2017"
+.TH FREEADDRINFO 3XNET "May 19, 2022"
 .SH NAME
 freeaddrinfo, getaddrinfo \- get address information
 .SH SYNOPSIS
-.LP
 .nf
 \fBcc\fR [ \fIflag\fR ... ] \fIfile\fR ... \fB-lxnet\fR [ \fIlibrary\fR ... ]
 #include <sys/socket.h>
@@ -64,7 +63,6 @@ freeaddrinfo, getaddrinfo \- get address information
 .fi
 
 .SH DESCRIPTION
-.LP
 The \fBfreeaddrinfo()\fR function frees one or more \fBaddrinfo\fR structures
 returned by \fBgetaddrinfo()\fR, along with any additional storage associated
 with those structures. If the \fBai_next\fR member of the structure is not
@@ -196,7 +194,6 @@ protocol; the value of \fBai_protocol\fR is interpreted as in a call to the
 \fBsocket()\fR function with the corresponding values of \fBai_family\fR and
 \fBai_protocol\fR.
 .SH RETURN VALUES
-.LP
 A 0 return value for \fBgetaddrinfo()\fR indicates successful completion; a
 non-zero return value indicates failure. The possible values for the failures
 are listed in the ERRORS section.
@@ -233,7 +230,6 @@ are not filled in through an explicit argument (for example,
 \fBsin6_flowinfo\fR) are set to 0, making it easier to compare socket address
 structures.
 .SH ERRORS
-.LP
 The \fBgetaddrinfo()\fR function will fail if:
 .sp
 .ne 2
@@ -287,7 +283,7 @@ return value.
 \fB\fBEAI_NONAME\fR\fR
 .ad
 .RS 16n
-he name does not resolve for the supplied parameters.  Neither \fInodename\fR
+The name does not resolve for the supplied parameters.  Neither \fInodename\fR
 nor \fIservname\fR were supplied. At least one of these must be supplied.
 .RE
 
@@ -328,7 +324,6 @@ An argument buffer overflowed.
 .RE
 
 .SH USAGE
-.LP
 If the caller handles only TCP and not UDP, for example, then the
 \fBai_protocol\fR member of the \fIhints\fR structure should be set to
 \fBIPPROTO_TCP\fR when \fBgetaddrinfo()\fR is called.
@@ -338,7 +333,6 @@ If the caller handles only IPv4 and not IPv6, then the \fBai_family\fR member
 of the \fIhints\fR structure should be set to \fBAF_INET\fR when
 \fBgetaddrinfo()\fR is called.
 .SH ATTRIBUTES
-.LP
 See \fBattributes\fR(7) for descriptions of the following attributes:
 .sp
 
@@ -355,7 +349,6 @@ MT-Level	MT-Safe
 .TE
 
 .SH SEE ALSO
-.LP
 .BR connect (3XNET),
 .BR gai_strerror (3XNET),
 .BR gethostbyname (3XNET),
-- 
2.42.0

